description: some libraries fail to link when --as-needed is set
author: Michael Gilbert <mgilbert@debian.org>

--- a/build/config/compiler/BUILD.gn
+++ b/build/config/compiler/BUILD.gn
@@ -385,7 +385,7 @@ config("compiler") {
     if (!using_sanitizer) {
       ldflags += [
         "-Wl,-z,defs",
-        "-Wl,--as-needed",
+        "-Wl,--no-as-needed",
       ]
     }
   }
